Jupyter Notebooks/LUVOIR/PASTIS_Modes_E2E.ipynb

```diff
@@ -0,0 +1,1143 @@
+{
+ "cells": [
+  {
+   "cell_type": "markdown",
+   "id": "foster-subdivision",
+   "metadata": {},
+   "source": [
+    "#  This notebook is for testing and integrating PASTIS_Modes_E2E.py with the current develop branch"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "id": "authentic-report",
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "import os\n",
+    "import time\n",
+    "import functools\n",
+    "from shutil import copy\n",
+    "import numpy as np\n",
+    "import matplotlib.pyplot as plt\n",
+    "from matplotlib.colors import LogNorm\n",
+    "from astropy.io import fits\n",
+    "import astropy.units as u\n",
+    "import hcipy as hc\n",
```

Comment:
No need to change it here, but let's not do this anymore. `hc` can mean so many things, let's just use `import hcipy` in the future.